Kms/owlbot.py

```diff
@@ -110,3 +110,34 @@
     r"(.{0,})\]\((/.{0,})\)",
     r"\1](https://cloud.google.com\2)"
 )
+
+# fix backwards-compatibility issues due to removed resource name helpers
+
+s.replace(
+    'src/V1/Gapic/KeyManagementServiceGapicClient.php',
+    r"}",
```

Comment:
This operation is not idempotent.

Can you grep the new function name and if it doesn't exist inject it?

Reply:
great idea! Is there a `grep` functionality in synthtool, or should I just implement it in python like this


```py
 f = open('src/V1/Gapic/KeyManagementServiceGapicClient.php',  "r")
   if 'public static function cryptoKeyName' not in f.read():
     s.replace(..)
```

Reply:
Okay, implemented in latest change. PTAL!